laos_gggi/data_functions/combine_data.py

```diff
@@ -21,6 +21,45 @@ def load_all_data():
     # 2. EM-DAT data representing the event damages (index: Year, ISO3)
     merged_dict["emdat_damage"] = emdat["df_inten_filtered_adjusted"]
 
+    emdat["df_inten_filtered_adjusted_hydro"] = emdat[
+        "df_inten_filtered_adjusted_hydro"
+    ].drop(columns=["Region", "Subregion"])
+    for x in emdat["df_inten_filtered_adjusted_hydro"].columns:
+        emdat["df_inten_filtered_adjusted_hydro"].rename(
```

Comment:
`columns` can be a function in rename:

```
emdat["df_inten_filtered_adjusted_hydro"] =( emdat
    .loc[:, "df_inten_filtered_adjusted_hydro"]
    .drop(columns=["Region", "Subregion"])
    .rename(columns = lambda x: f'{x}_clim')
)
```

Reply:
Nice! The lambda functions are a super power I need to work on. 

I implemented it with a small change, as emdat is a dictionary and does not have .loc[]

emdat["df_inten_filtered_adjusted_clim"] =  (emdat[
    "df_inten_filtered_adjusted_clim"
].drop(columns=["Region", "Subregion"]).rename(columns = lambda x: f'{x}_clim'))